src/vs/workbench/browser/parts/views/treeView.ts

```diff
@@ -1183,3 +1214,30 @@ export class CustomTreeView extends TreeView {
 		}
 	}
 }
+
+export class CustomTreeViewDragAndDrop implements ITreeDragAndDrop<ITreeItem> {
+	constructor(private dataSource: TreeDataSource) { }
+
+	onDragOver(data: IDragAndDropData, targetElement: ITreeItem, targetIndex: number, originalEvent: DragEvent): boolean | ITreeDragOverReaction {
+		return { accept: true, bubble: TreeDragOverBubble.Down, autoExpand: true };
```

Comment:
What does `accept` do here?

Reply:
If the accept option is true then the target tree item gets highlighted (this is the same behavior as the drag and drop in the file explorer).

Reply:
Makes sense, thanks!